frontend/src/main/components/SectionsInstructorTableBase.js

```diff
@@ -0,0 +1,78 @@
+import React, { Fragment } from "react";
+import { useTable, useGroupBy, useExpanded } from "react-table";
+import { Table } from "react-bootstrap";
+
+// Stryker disable StringLiteral, ArrayDeclaration
+export default function SectionsInstructorTableBase({
+  columns,
+  data,
+  testid = "testid",
+}) {
+  // Stryker disable next-line ObjectLiteral
+  const { getTableProps, getTableBodyProps, headerGroups, rows, prepareRow } =
+    useTable(
+      {
+        initialState: {
+          groupBy: [],
+          hiddenColumns: ["isSection"],
+        },
+        columns,
+        data,
+      },
+      useGroupBy,
+      useExpanded,
+    );
+
+  return (
+    <Table {...getTableProps()} striped bordered hover>
+      <thead key="thead">
```

Comment:
No need for stable keys in statically defined elements.
```suggestion
      <thead>
```

Reply:
Resolved!